README.md

```diff
@@ -43,3 +43,15 @@ If there is not SITL instance attached, these keys can be used to control the RO
     Before opening Bluesim (it launches its own SITL instance), run:
 
  `sim_vehicle.py -j6 -L RATBeach --frame JSON --out=udpout:0.0.0.0:14550`
+
+# External Levels
+
+External levels can be loaded by placing a .pck file in the "levels" folder at the same level as the simulator executable (the folder is created at first run if it does not exist).
```

Comment:
That's not happening, the folder does not exist after the simulator runs for the first time, for test, I just exported a linux binary and it's failing to create the levels folder.

Reply:
can you try now?